tests/UnitTests/CoreGraphics.drawing/CGContextDrawing_PatternTests.cpp

```diff
@@ -0,0 +1,373 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "DrawingTest.h"
+
+#pragma region Stencil Pattern
+
+static void drawStencilStar(void* info, CGContextRef context) {
+    double r = 0.8 * 16 / 2;
```

Comment:
this is just a test, doesn't make sense to make everything in tests as consts.
if so  paths test would have done mad :).


---
In reply to: [101328691](https://github.com/Microsoft/WinObjC/pull/1952#discussion_r101328691) [](ancestors = 101328691)